package.json

```diff
@@ -195,6 +195,7 @@
 		"eslint-plugin-import": "2.25.2",
 		"eslint-plugin-jest": "27.2.3",
 		"eslint-plugin-jest-dom": "5.0.2",
+		"eslint-plugin-prettier": "5.0.0",
```

Comment:
I don't think we need it anymore here after upgrading the lock file. Although if there is a reason to keep it, it's fine.

Reply:
I had to add this to the root `package.json` because otherwise the plugin got installed only into the `/packages/eslint-plugin/node_modules` subdirectory, not to the root `node_modules`. For some reason, even the new `npm` client didn't hoist the package, although it could, the slot is available, there is no second version of the same package that would conflict. And the `wp-scripts lint-js` command fails when the ESLint plugins are not resolvable from the root directory.

Reply:
Based on my experience, it's an issue with npm in general. After the lock file gets correctly updated, if you remove the dependency from `package.json`, it should remain hoisted. However, I don't guarantee it. It works in most of the cases 😄